src/components/Header/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { NavLink } from 'react-router-dom';
+// import MenuIcon from '@mui/icons-material/Menu';
```

Comment:
every place u have comment out component or block of code u can delete it if its not in use